release notes/v0.27.0.md

```diff
@@ -0,0 +1,81 @@
+
+These new scenarios are entirely optional, and existing k6 scripts and options should continue to work the same as before (with a few minor breaking changes, see below), but please [create a bug issue](https://github.com/loadimpact/k6/issues/new?labels=bug&template=bug_report.md) if that's not the case.
+
+See the [documentation](https://k6.io/docs/using-k6/executors) for details and examples, or keep reading for the summary.
+
+#### Execution scenarios
+
+Some of the currently possible scenarios were formalized into standalone executors:
+
+- Shared iterations: a fixed number of iterations are "shared" by all VUs, and the test ends once all iterations are executed. This executor is equivalent to the global `vus` and `iterations` options.
+- Constant VUs: a fixed number of VUs execute as many iterations as possible for a specified amount of time. This executor is equivalent to the global `vus` and `duration` options.
+- Ramping VUs: a variable number of VUs execute as many iterations as possible for a specified amount of time. This executor is equivalent to the global `stages` option.
+- Externally controlled: control and scale execution at runtime via k6's REST API or the CLI.
+
+And new executors were added to support some of the most frequently requested scenarios that were previously difficult or impossible to model:
+
+- Per VU iterations: each VU executes a fixed number of iterations (#381).
+- Constant arrival rate: a fixed number of iterations are executed in a specified period of time. This allows k6 to dynamically change the amount of VUs during a test run to achieve the specified amount of iterations per period, which is useful for a more accurate representation of RPS, for example. See #550 for details.
+- Ramping arrival rate: a variable number of iterations are executed in a specified period of time. This is similar to the ramping VUs executor, but for iterations instead.
+
+#### Execution improvements
+
+- Several executors can be configured in a single test run, both sequentially and in parallel (closes #239).
+- Each executor has a `startTime` property, which defines at what time relative to the test run start it will start executing.
+- Executors have a new `gracefulStop` property that allows for iterations to complete gracefully for some amount of time after the normal executor duration is over (closes #879); even with `gracefulStop=0`, i.e. the old behavior, metrics are emitted and processed in a much saner way (closes #1033).
+- The new `--execution-segment` and `--execution-segment-sequence` options allow easy partitioning of test runs across multiple k6 instances. Initially this applies to configuration options like number of VUs or iterations, but opens the door to test data partitioning, an often requested feature. See #997 for more details.
```

Comment:
You mean moving it as a paragraph after the list? This was originally a list of "Execution improvements", but since we added the example, I guess it doesn't make sense anymore. The same goes for the new `dropped_iterations` item, I guess.

Not sure, let me know what you prefer. :)